### ISSUE TITLE
Feature Request: Scroll Wheel Changes Text and Shape Size in image editor

### ISSUE THREAD

**Issue by abegertler** [ShareX/ShareX#7657]
For transparency I am copying this from Feature Request #6890 (which was marked as stale) to bring it back up, because they put it perfectly!
For me, this is the only feature still missing from ShareX!! And otherwise I really love it. Thank you.

"
I recently switched from Lightshot to ShareX and I am loving ShareX.

However, as someone who annotates dozens of images a week, I loved how in Lightshot that the mouse wheel changed the size of text, arrows, box boarders and objects when editing an image after a screenshot.

In ShareX, the mouse wheel just changes the magnifier size which is not as frequent a use case.

I would be thrilled with a faster way to change the size of text, arrows and objects in ShareX's image editor. I suggest using the scroll wheel like Lightshot as the user experience is already very intuitive and super useful.
"

**Comment by LoganTraceur** [comment]
Yes please! almost 2 years on and this is still my only real frustration with shareX.
I would love to see this implemented, even as an optional setting that could be toggled on.
